### ISSUE TITLE
Merge upstream?

### ISSUE THREAD

**Issue by vgarleanu** [ThaUnknown/jassub#22]
Really awesome project! I'm really glad someone forked JSO!

That said, I was wondering, are there any plans on upstreaming the changes and performance fixes in this fork?

**Comment by GizmoTjaz** [comment]
https://github.com/libass/JavascriptSubtitlesOctopus/pull/138
https://github.com/libass/JavascriptSubtitlesOctopus/pull/128

**Comment by ThaUnknown** [comment]
no, jassub is not something I'd consider a fork, considering i rewrote 90% of the lib from scratch, jso devs are also hell bent on some weird PR structures, which are a massive waste of time so they can go do one, i tried merging improvements but they are unreasonable

you have very litte reason to use JSO over JASSUB, JSO is slower, buggier, larger, supports less features etc etc, and everyone who used it knows that, which is why pretty much all major repositories migrated from JSO to JASSUB, including vidstack, jellyfin and others, just dim is left, but the PR is pending
